infra/modules/database/main.tf

```diff
@@ -304,6 +304,10 @@ resource "aws_lambda_function" "role_manager" {
   # checkov:skip=CKV_AWS_116:Dead letter queue (DLQ) configuration is only relevant for asynchronous invocations
 }
 
+output "role_manager_function_name" {
+  value = aws_lambda_function.role_manager.function_name
+}
+
```

Comment:
can you move this block to a new file `/infra/modules/database/outputs.tf`

Reply:
Yep, updated in https://github.com/navapbc/template-infra/pull/368/commits/2318beccb30288f521b7dace901e5bf56f296f2e